### ISSUE TITLE
Addbooks Python3 Regression: Bytes & Strings /edit + addbook

### ISSUE THREAD

**Issue by mekarpeles** [internetarchive/openlibrary#3808]
@cclauss this is a hotfix for /editing. @cdrini and I both notice these changes won't work on Python 3. Opening a separate issue to unblock prod

_Originally posted by @mekarpeles in https://github.com/internetarchive/openlibrary/pull/3807#issuecomment-693620518_

**Comment by cclauss** [comment]
Fixed in #3931
